### PR TITLE
fast follow design edits

### LINKED ISSUE #9237: Validation Tool | Post-Launch Design/Content QA
## Problem statement

The design and content team needs to complete thorough QA of the File Validator tool post-launch to ensure the feature is in-line with the design and user experience intent. 

## What you need to know

- [Figma Prototype](https://www.figma.com/proto/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?node-id=2183-21892&scaling=min-zoom&page-id=2173%3A22160&starting-point-node-id=2183%3A21892)
-  [Figma Workspace](https://www.figma.com/file/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?node-id=2173%3A22160&t=TDYOsF1m6itOtDFD-1) for comp matching; toggle on "spacing-guide" layers for Vertical Spacing Style Guide

## Acceptance criteria
New or existing tickets address the following:
- [x] BUG: The "Continue Without Changes" button appears when Errors are present. It should ONLY appear if the file only returns Warnings. 
- [x] DESIGN: Switch the button style of "Try Another File" to be the PRIMARY Style and not the OUTLINE Style
- [x] DESIGN: Comp match vertical spacing throughout
- [x] DESIGN: Comp match Tip Callout (border styling, rounded corner styling, icon & text offset padding)
- [x] DESIGN: Comp match "Download as CSV" button (resolve icon size & alignment)
- [x] DESIGN: Comp match size of "Warnings" and "Errors" headers 
- [ ] DESIGN: Comp match tight padding of Warnings and Errors tables 
- [x] CONTENT: Edit file input CTA for brevity: "Make sure your file has a .csv extension"

- [x] CONTENT and DESIGN: In Production. If there is a Jurisdictional error, the warning label is a red box above the Jurisdicational Error Table, but it needs to instead be the Yellow block as shown on the Figma image. Location remains the same. 

## To do

- [ ] Engineering, product, and delivery reviews this ticket async
- [ ] Design and engineering check in on questions/comments (sync or async)
- [ ] Design, engineering, and product align on fix priorities
- [ ] Engineering tickets created, if needed

### PR DESCRIPTION
**Pull request by etanb**
This PR contains all the Design Fast Follows to our File Validator tool listed [here in 9237](https://app.zenhub.com/workspaces/web-experience-607d9d5e68b95200150fec37/issues/gh/cdcgov/prime-reportstream/9237).

New or existing tickets address the following:
- [x] BUG: The "Continue Without Changes" button appears when Errors are present. It should ONLY appear if the file only returns Warnings. 
- [x] DESIGN: Switch the button style of "Try Another File" to be the PRIMARY Style and not the OUTLINE Style
- [x] DESIGN: Comp match vertical spacing throughout
- [x] DESIGN: Comp match Tip Callout (border styling, rounded corner styling, icon & text offset padding)
- [x] DESIGN: Comp match "Download as CSV" button (resolve icon size & alignment)
- [x] DESIGN: Comp match size of "Warnings" and "Errors" headers 
- [x] DESIGN: Comp match tight padding of Warnings and Errors tables 
- [x] CONTENT: Edit file input CTA for brevity: "Make sure your file has a .csv extension"

- [x] CONTENT and DESIGN: In Production. If there is a Jurisdictional error, the warning label is a red box above the Jurisdicational Error Table, but it needs to instead be the Yellow block as shown on the Figma image. Location remains the same. 

Fixes #9237

### CODE REVIEW

**Comment by jillian-hammer** [comment]
On the last confirmation screen, add a hyperlink to the `API Programmer's Guide`. 

The text should read:

> Learn more about the onboarding process in the [API Programmer's Guide](https://reportstream.cdc.gov/resources/programmers-guide).

**Comment by jillian-hammer** [comment]
Jurisdictional errors are still showing the old alert style. The  jurisdictional error alert should take the place of the alert at the top of the page. 

CDC-Prime-ReportStream-TRIALFRONTEND01

**Comment by jillian-hammer** [comment]
Remove extraneous right padding (highlighted in orange) to ensure the button is right-aligned with the rest of the page content

image

**Comment by jillian-hammer** [comment]
In Warning-only cases, the `Test another file button` should be in the outline style and Contine without changes should be in the primary (filled) style. 

Build:
image

Design: 
image

**Comment by jillian-hammer** [comment]
Body text size & line-height on the confirmation screen are inconsistent with the text styling on the previous screens ("Check that public health departments can receive your data ..."). (It looks too small and needs some extra line spacing.) Can you take a look at making it consistent? 

image

**Comment by jillian-hammer** [comment]
Support call-out

- [ ] Fix typo from "Question" to -> "Questions" (should read "Questions or feedback?") 
- [ ] Text color should be base-darker (#3D4551)

Build:
image

**Comment by jillian-hammer** [comment]
Caught a typo here – remove extra "s" after "Resolve"

Should read:

> Your file does not meet the data model for the following jurisdiction(s). Resolve the errors below to ensure those jurisdictions can receive your data.

@etanb 

image

File: frontend-react/src/components/FileHandlers/FileHandlerProgrammersGuideTip.tsx [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
-                successfully. Pay special attention to which fields are required
-                and common mistakes.
-            </span>
-        </SiteAlert>
+        <section
+            data-testid="siteAlert"
+            className="usa-site-alert usa-site-alert--info usa-site-alert--no-heading usa-site-alert--no-icon border-primary-lighter border-width-1px border-solid radius-md"
+            aria-label="Site alert"
+        >
+            <div className="usa-alert">
```
Comment (penny-lischer) [inline frontend-react/src/components/FileHandlers/FileHandlerProgrammersGuideTip.tsx:12]: Were three <div> tags necessary? Can we just combine them into one?

**Review by jpandersen87 — APPROVED** [review]
I have the same question as @penny-lischer but otherwise seems good.

File: frontend-react/src/components/FileHandlers/FileHandlerProgrammersGuideTip.tsx (hunk 1 above)
Comment (etanb) [reply frontend-react/src/components/FileHandlers/FileHandlerProgrammersGuideTip.tsx]: good catch! using the raw USWDS component over the Trussworks one, and yes, without changing the UI, combining those 3 `<div>` with their associated classes works fine.

**Review by penny-lischer — APPROVED** [review]
LGTM

**Pull request merged**